hist/hist/inc/TAxis.h

```diff
@@ -75,6 +76,8 @@ class TAxis : public TNamed, public TAttAxis {
    TAxis();
    TAxis(Int_t nbins, Double_t xmin, Double_t xmax);
    TAxis(Int_t nbins, const Double_t *xbins);
+   TAxis(const std::vector<double>& bins);
+   TAxis(std::vector<double>&& bins);
```

Comment:
```suggestion
   TAxis(std::span<const double> bins);
```
I don't think the interface with the `std::vector` would age well. What about using `std::span`?

You can use it even when building ROOT with older C++ standards because of the ROOT internal `std::span` backport:
```c++
#include <ROOT/RSpan.hxx>
```

Reply:
Let's leave it with the `std::vector`. Otherwise, the initializer list would not work.